packages/data/src/plugins/persistence/index.js

```diff
@@ -193,7 +193,9 @@ function persistencePlugin( registry, pluginOptions ) {
 					//   subset of keys.
 					// - New keys in what would otherwise be used as initial
 					//   state are deeply merged as base for persisted value.
-					initialState = merge( {}, initialState, persistedState );
+					initialState = deepmerge( initialState, persistedState, {
+						isMergeableObject: isPlainObject,
```

Comment:
In theory, `obj => isPlainObject( obj )` would be safer here, because it ensures we're ever only passing one parameter into `isPlainObject`.

In practice, though, I think this is fine, as `isMergeableObject` and `isPlainObject` both currently use a single parameter, and would both have to change their method signatures to have incompatible 2nd parameters for this to ever become a problem.

Reply:
Yep, I think it's very unlikely for both signatures to change. Good observation though 👍